wikichange/src/popup/popup.html

```diff
@@ -5,22 +5,22 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="./assets/popup.css" />
     <title>Wiki Addition Viewer</title>
   </head>
   <body>
     <div class="card" style="width: 18rem; height: 5rem;">
-      <div class="alert alert-secondary" role="alert">
+      <div class="alert alert-secondary container" role="alert">
         <div class="text-center">
           <h4 class="alert-heading">Wiki Addition Viewer</h4>
           <p>Welcome to our comps project: An extension to add insights to your Wikipedia experience.</p>
           <a href="https://sukritsangvong.github.io/wp-browser-extension/" id="comps-website" class="btn btn-secondary btn-sm">Learn more!</a>
           <hr>
         </div>
-        <p>
           <ul>
             <li>Year: 2022-2023</li>
           </ul>
-        </p>
+          <a role="button" href="" id="options">Extension Options</a>
```

Comment:
Maybe center year too? It's a little bit weird as it is to have one at the center and another one being a bullet